iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/Tools.java

```diff
@@ -136,20 +139,22 @@ public synchronized static Device addDeviceWithRetry(RegistryManager registryMan
         return ret;
     }
 
-    public synchronized static Module addModuleWithRetry(RegistryManager registryManager, Module module) throws IotHubException, IOException, InterruptedException
```

Comment:
Same as above. This synchronization placed a bottle neck on the SDK tests, so I'm removing it